lib/Doctrine/ORM/Tools/Console/Command/ConvertMappingCommand.php

```diff
@@ -75,7 +75,13 @@ protected function configure()
 by the ORM, you can use a DBAL functionality to filter the tables and sequences down
 on a global level:
 
-    $config->setFilterSchemaAssetsExpression($regexp);
+    $config->setSchemaAssetsFilter(function (string|AbstractAsset $assetName) use ($regexp): bool {
+        if ($assetName instanceof AbstractAsset) {
+            $assetName = $assetName->getName();
+        }
+
+        return (bool) preg_match($regexp, $assetName);
```

Comment:
I don't think we should stick with a regexp example:

- it involves an undefined variable, which we pass via `use`;
- it involves a boolean cast.

Can you please use something more simple, like maybe `in_array`, or `str_starts_with`?

Reply:
@greg0ire done, take a look again, please